hack/scripts-dev/docker-dns/certs-wildcard/run.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+rm -rf /tmp/m1.data /tmp/m2.data /tmp/m3.data
+
+/etc/init.d/bind9 start
+
+# get rid of hosts so go lookup won't resolve 127.0.0.1 to localhost
+cat /dev/null >/etc/hosts
+
+goreman -f /certs-wildcard/Procfile start &
+sleep 7s
```

Comment:
why sleep?

Reply:
sleep in code always seems hacky.

Reply:
You need sleep until cluster is ready in this case.

Reply:
hmm, it seems like this test can be flaky since we don't really know when etcd cluster will be ready. is etcd cluster is rdy at 30s, the test will fail right?

Reply:
We don't run these in CIs anyway. For now, I just manually tested it and confirm that it works. Later when we integrate with CI, we can add some timeouts.

Reply:
@gyuho maybe add a TODO here to create some sort of waiting logic with timeout. if etcd cluster is ready, then go ahead. if not and has timeout, exit with error.